pegasus/data/csv.rb

```diff
@@ -8,7 +8,8 @@ class CsvToSqlTable
   def initialize(path, params={})
     @db = params[:db] || PEGASUS_DB
     @path = path
-    @table = File.basename(@path, File.extname(@path)).tr('-', '_').to_sym
+    table_prefix = params[:table_prefix] || ''
+    @table = (table_prefix + File.basename(@path, File.extname(@path)).tr('-', '_')).to_sym
```

Comment:
please move this file to somewhere in `lib/cdo`, and make both dashboard and pegasus pass in the DB as a parameter. the reason is that the goal of this swarm is to separate pegasus and dashboard code in a way that makes it clear that neither is accessing the other's DB.

Reply:
I removed that param bag to be more explicit but I will move the file on a different commit.

I tried but got a bit verbose. The file name is csv and we already have a csv file. Then it needed remapping some paths. I decided to deal with it as a separate mini problem.